ios/FluentUI.Demo/FluentUI.Demo/Demos/NotificationViewDemoController_SwiftUI.swift

```diff
@@ -51,22 +52,23 @@ struct NotificationDemoView: View {
         let messageButtonAction = hasMessageAction ? { showAlert = true } : nil
         let hasMessage = !message.isEmpty
         let hasTitle = !title.isEmpty
-        let notification = FluentNotification(style: style,
-                                              isFlexibleWidthToast: isFlexibleWidthToast,
-                                              message: hasMessage ? message : nil,
-                                              attributedMessage: hasAttribute && hasMessage ? attributedMessage : nil,
-                                              title: hasTitle ? title : nil,
-                                              attributedTitle: hasAttribute && hasTitle ? attributedTitle : nil,
-                                              image: image,
-                                              trailingImage: trailingImage,
-                                              trailingImageAccessibilityLabel: showTrailingImage ? "Circle" : nil,
-                                              actionButtonTitle: actionButtonTitle,
-                                              actionButtonAction: actionButtonAction,
-                                              messageButtonAction: messageButtonAction)
-                            .overrideTokens(overrideTokens ? NotificationOverrideTokens() : nil)
 
         VStack {
-            notification
+            Rectangle()
```

Comment:
Why do we use this instead of the notification variable?

Reply:
Now that the notification has its opacity set to see-through when it isn't presented, the other notification wasn't visible. Since it wasn't presented. So I added a placeholder Rectangle to present the preview notification in.